scripts/performance/perf_load/perf_client_fees.py

```diff
@@ -45,7 +47,9 @@ def __init__(self, name, pipe_conn, batch_size, batch_rate, req_kind, buff_req,
         self._plugin_lib = kwargs.get("plugin_lib", "")
         self._plugin_init = kwargs.get("plugin_init", "")
         self._req_num_of_trustees = kwargs.get("trustees_num", 4)
-        self._set_fees = kwargs.get("set_fees", {})
+        fees = kwargs.get("set_fees", {})
+        self._auth_rule_metadata = {k: {"fees": random_string()} for k, _ in fees.items()}
+        self._set_fees = {self._auth_rule_metadata.get(k, {}).get('fees', None): v for k, v in fees.items()}
```

Comment:
I think it makes sense to assume that `[k]['fees']` is inside the dict or use one common loop to set both `self._auth_rule_metadata` and `self._set_fees` at once.